docs/source/concepts/PeaksWorkspace.rst

```diff
@@ -16,16 +16,18 @@ Creating a PeaksWorkspace
 * :ref:`CreatePeaksWorkspace <algm-CreatePeaksWorkspace>` will create an empty PeaksWorkspace that you can then edit.
 
 Viewing a PeaksWorkspace
---------------------------
+------------------------
 
 * Double-click a PeaksWorkspace to see the full list of data of each Peak object.
-* In MantidPlot, you can drag/drop a PeaksWorkspace from the list of workspaces onto the `Instrument View <http://www.mantidproject.org/MantidPlot:_Instrument_View>`__ . This will overlay the peaks onto the detector face.
-* `PeaksViewer <http://www.mantidproject.org/PeaksViewer>`__ in the `SliceViewer <http://www.mantidproject.org/SliceViewer>`__
+* In MantidWorkbench, you can drag/drop a PeaksWorkspace from the list of workspaces onto the
+:ref:`Instrument View <InstrumentViewer>`. This will overlay the peaks onto the detector face.
+* `PeaksViewer <http://www.mantidproject.org/PeaksViewer>`__ in the
+`SliceViewer <http://www.mantidproject.org/SliceViewer>`__.
```

Comment:
```suggestion
* In MantidWorkbench, you can drag/drop a PeaksWorkspace from the list of workspaces onto the
  :ref:`Instrument View <InstrumentViewer>`. This will overlay the peaks onto the detector face.
* `PeaksViewer <http://www.mantidproject.org/PeaksViewer>`__ in the
  `SliceViewer <http://www.mantidproject.org/SliceViewer>`__.
```

I think this will fix the build failure!